make/Defs.gmk

```diff
@@ -111,7 +111,7 @@ JDKJAVA = $(JDKHOME)/bin/java
 JDKJAVAC = $(JDKHOME)/bin/javac
 JAR = $(JDKHOME)/bin/jar
 
-AGENT_JAVAC_SOURCE_TARGET = --release 8
+AGENT_JAVAC_SOURCE_TARGET = --release 11
```

Comment:
No. The use of 8 is deliberate, so that we can run use jtreg to test older versions of JDK.

Reply:
OK. Reverting to `--release 8` and replacing all `var` usages with explicit type references.

Reply:
Maybe prefix the line with a comment of the form:
```
// Specify oldest supported JDK version that can be tested
```